packages/ckeditor5-image/src/imageutils.ts

```diff
@@ -83,7 +84,7 @@ export default class ImageUtils extends Plugin {
 	 * @param attributes Attributes of the inserted image.
 	 * This method filters out the attributes which are disallowed by the {@link module:engine/model/schema~Schema}.
 	 * @param selectable Place to insert the image. If not specified,
-	 * the {@link module:widget/utils~findOptimalInsertionRange} logic will be applied for the block images
+	 * the {@link module:engine/model/schema~Schema#findOptimalInsertionRange} logic will be applied for the block images
```

Comment:
This was using the widget implementation of this method. I think this should not change.

Reply:
Adressed in 24ac1fa